pkgs/by-name/av/av1an-unwrapped/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  av1an-unwrapped,
+  pkg-config,
+  ffmpeg,
+  nasm,
+  vapoursynth,
+  libaom,
+  rav1e,
+}:
+rustPlatform.buildRustPackage {
+  pname = "av1an-unwrapped";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "master-of-zen";
+    repo = "av1an";
+    rev = av1an-unwrapped.version;
+    hash = "sha256-A4/1UdM8N5CHP44PBNB+ZH2Gcl84rcpUBwQRSnubBGc=";
+  };
+
+  cargoHash = "sha256-ahoiCAUREtXgXLNrWVQ2Gc65bWMo4pIJXP9xjnQAlaI=";
+
+  nativeBuildInputs = [
+    rustPlatform.bindgenHook
+    pkg-config
+    nasm
+  ];
+
+  buildInputs = [
+    ffmpeg
+    vapoursynth
+  ];
+
+  nativeCheckInputs = [
+    libaom
+    rav1e
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

tracking issue: #292468
discouraged as it leads to ambiguous `with`s